### DISCUSSION TITLE
Red eclipse viewmodels changes (non-red eclipse images)

### DISCUSSION THREAD

**Discussion by thevibingcat** [redeclipse/base#1357] [Ideas and Feature Requests]
Red eclipse is often a competitive game, so I think there should be more minimized viewmodels this new feature should also be toggleable, so people who like the default models can keep them.
The purpose of this new feature would be so you can see more of your screen
minimized viewmodels examples from Team Fortress 2:
(without min viewmodels):
![Screenshot from 2022-07-16 17-10-47](https://user-images.githubusercontent.com/102605902/179372186-0a919807-8454-4afc-ba71-06a81ce3f5de.png)
(with min viewmodels):
![Screenshot from 2022-07-16 17-11-08](https://user-images.githubusercontent.com/102605902/179372188-452a44a6-3b79-4a53-bb3a-807edec4f365.png)

**Comment by SniperGoth** [comment]
There is already an option to disable first person arms and models, you need to open the console and type:
/firstpersonmodel 0, which will take out both the arms and legs.
/firstpersonmodel 1, which will only have the arms, not the legs.
/firstpersonmodel 2, which will only have legs, and no arms.
/firstpersonmodel 3, which has both arms and legs into view, and is on by default.

As for actually making them small, i don't quite remember now if there is a way to do it without messing with the cfg files themselves.
Personally i use firstpersonmodel 1, which has no legs, but that's up to personal preference.
![20220716_182707](https://user-images.githubusercontent.com/24359960/179372531-4627a449-efc1-4925-ba56-f72d5173b0cd.png)

And this is 0, without anything in display.
![20220716_182716](https://user-images.githubusercontent.com/24359960/179372533-43578a43-d5ae-43f3-9c58-ba89211815d3.png)

**Comment by Calinou** [comment]
https://github.com/redeclipse/base/pull/1324 would provide what you want here. In the meantime, you can use `/firstpersonblend 0.5` to make first-person models translucent.
